liquibase-core/src/main/java/liquibase/datatype/core/SmallIntType.java

```diff
@@ -61,7 +61,9 @@ public DatabaseDataType toDatabaseDataType(Database database) {
             }
             return new DatabaseDataType("SMALLINT"); //always smallint regardless of parameters passed
         }
-
+        if (database instanceof SybaseDatabase) {
```

Comment:
Should the SybaseASADatabase also be included in this logic?

Reply:
I'm not sure about smallint vs. tinyint. I didn't go hunting through their docs on what datatypes they support and was just reviewing this PR for the fixes it specifically had.